sds_data_manager/stacks/networking_stack.py

```diff
@@ -61,6 +61,13 @@ def __init__(
             self, "FargateInstanceSecurityGroup", vpc=self.vpc
         )
 
+        self.vpc.add_interface_endpoint(
```

Comment:
Could you add a comment about why this is added. Is it that our public lambdas need to access our secrets manager when getting the rds secret to query the db?